interfaces/apparmor/backend.go

```diff
@@ -103,7 +103,26 @@ func (b *Backend) Setup(snapInfo *snap.Info, developerMode bool, repo *interface
 	if errReload != nil {
 		return errReload
 	}
-	return errUnload
+	if errUnload != nil {
+		return errUnload
+	}
+	// Sanity checking, see if all the profiles are really loaded and load
+	// those profiles that are not. This can happen if the files were on disk
+	// but were not really loaded (e.g. due to an interrupted install).
```

Comment:
Without patching apparmor to give us cryptographic hashes of the loaded profile we are always going to be in a hard place. We could unload all profiles but that will quickly bite us as those things are slow to compute.

@jdstrand What do you think?